packages/rs-drive/src/drive/identity/withdrawals/documents.rs

```diff
@@ -11,15 +11,16 @@ use grovedb::TransactionArg;
 use indexmap::IndexMap;
 
 use crate::drive::document::query::QueryDocumentsOutcomeV0Methods;
+use crate::drive::identity::withdrawals::WithdrawalTransactionIndex;
 use crate::{
     drive::Drive,
     error::{drive::DriveError, Error},
     query::{DriveQuery, InternalClauses, OrderClause, WhereClause},
 };
 
 impl Drive {
-    /// Fetch withdrawal documents by it's status
-    pub fn fetch_withdrawal_documents_by_status(
+    /// Fetch withdrawal documents by it's status ordered by updated_at ascending with limit 100
+    pub fn fetch_up_to_100_oldest_withdrawal_documents_by_status(
```

Comment:
maybe put the limit as a parameter? Also this should be versioned.

Reply:
You are thinking of making this param optional such as we have in DriveQuery? I put 100 into the name of the function to highlight the limitation.

Reply:
Yes but why 100? There's no reason to get 100, in the document we we were supposed to get 16 I think.

Reply:
This function is used to change the state of broadcasted documents, so I think limitation 100 is good there. It also used in tests a lot where we need all documents so theoretically it should work without limit at all but currently test are written in the way that it won't be more than 100 expected elements

Reply:
please make it a variable number.